lib/wda/webdriveragent.js

```diff
@@ -96,7 +96,21 @@ class WebDriverAgent {
     }
   }
 
+  /**
+   * Return boolean if WDA is running or not
+   * @return {boolean} True if WDA is running
+   * @throws {Error} If there was invalid response code or body
+   */
   async isRunning () {
```

Comment:
I'd rather create a separate method for `getStatus()` and then call it inside `getRunningBundleId` and `isRunning`